.github/workflows/tf_apply_production.yml

```diff
@@ -66,3 +66,7 @@ jobs:
       - name: Apply cloudfront 
         working-directory: terragrunt/env/production/cloudfront
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
+
+      - name: Apply dynamodb 
+        working-directory: terragrunt/env/production/dynamodb
+        run: terragrunt apply --terragrunt-non-interactive -auto-approve
```

Comment:
Could you move this to before the `api` step since it will depend on the outputs of this module.

Reply:
I changed the order